src/views/AdminActivityEdit.vue

```diff
@@ -176,6 +355,9 @@ export default {
         key => this.activities[key].activityName === this.activityName
       );
       console.log({ key: this.currentKey });
+    },
+    removeKeyword(deleteKeyword) {
+      this.activity.activityKeywords = this.activity.activityKeywords.split(",").filter((keyword) => keyword !== deleteKeyword).join(",");
```

Comment:
could use the `this.keywords` computed function here

Reply:
good point!

Reply:
fixed